scripts/exglobal_atmos_analysis.sh

```diff
@@ -466,7 +466,7 @@ ${NLN} ${ESIASI}           iasibufrears
 ${NLN} ${IASIDB}           iasibufr_db
 ${NLN} ${AMSREBF}          amsrebufr
 ${NLN} ${AMSR2BF}          amsr2bufr
-${NLN} ${GMI1CRBF}         gmibufr
+#${NLN} ${GMI1CRBF}         gmibufr # GMI temporarily disabled due to array overflow.
```

Comment:
@emilyhcliu , @ADCollard , @HaixiaLiu-NOAA, @xincjin-NOAA.   What GMI behavior do you want from the `develop` g-w?  Retain, skip, or something else?

Reply:
Thanks for checking on this with other DA folks @RussTreadon-NOAA !

Reply:
Sorry for the later response, I though other persons might response. I would suggest to skip it for now, because it does not work properly.

Reply:
Thanks @xincjin-NOAA for the feedback. Based on your reply I will leave this line commented out as it is in this PR (e.g. skip it). We can uncomment it in the future should it be fixed.

Reply:
Added note in https://github.com/NOAA-EMC/global-workflow/issues/1356 to uncomment line when ingesting upcoming ops updates.